packages/file_selector/file_selector/lib/file_selector.dart

```diff
@@ -9,7 +9,19 @@ import 'package:file_selector_platform_interface/file_selector_platform_interfac
 export 'package:file_selector_platform_interface/file_selector_platform_interface.dart'
     show XFile, XTypeGroup;
 
-/// Open file dialog for loading files and return a file path
+/// Open file dialog for loading files and return a file path.
+///
+/// [acceptedTypeGroups] is the file type that can be selected in the dialog.
+/// there are differences in behavior depending on the platform as follows.
```

Comment:
The `.` should be a `:` here, and this is not capitalized.

Reply:
Also, this shouldn't be presented an exhaustive list, because other implementations will be added later. It should say something more like:
```
/// How this will be interpreted depends on the platform. For example:
/// - On Windows, each group will be an entry in a list of filter options.
/// - On macOS, the union of all types allowed by all of the groups will be allowed.
```